### PR TITLE
add pyvista 2D example plotting script

### PR DESCRIPTION
**Pull request by naliboff**
This PR adds an example for how to use pyvista (python-based plotting packages) for loading and plotting aspect pvtu files. 

While the workflows for making a 2D plot are actually a bit complicated as PyVista is inherently designed for making 3D plots, it can work and has been used in workflows to make publication-quality figures (see documentation). 

Perhaps more importantly, this provides a quick way to get ASPECT vtk files into python for further analysis and, if desired, make plots via a scripting interface.

The code in this PR creates the following figure:
pyvista_2D_example

### For all pull requests:

* [X] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](https://aspect-documentation.readthedocs.io/en/latest/user/extending/testing/writing-tests.html) for the new feature/benchmark in the [tests/](../blob/main/tests/) directory.
* [ ] I have added a changelog entry in the [doc/modules/changes](../blob/main/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

**Comment by naliboff** [comment]
@anne-glerum @bobmyhill - Could you take a look at this when you have a chance and assess whether this would be a useful example to have in the python contrib folder?

File: contrib/python/scripts/pyvista_2d_example.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+# Plot the image with imshow
+ax.imshow(img,aspect='equal',extent=plot_spatial_bounds)
+plt.tight_layout()
+
+# Save the python plot
+plt.savefig('pyvista_2d_example.png')
+
+# Close the figure
+plt.close()
```
Comment (bobmyhill) [inline contrib/python/scripts/pyvista_2d_example.py:106]: new line needed

File: contrib/python/scripts/pyvista_2d_example.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+###
+
+# Create the image via a screenshot
+img = plotter.screenshot(transparent_background=True,return_img=True)
+
+# Create the plot axis
+ax = plt.gca()
+
+# Plot the image with imshow
+ax.imshow(img,aspect='equal',extent=plot_spatial_bounds)
```
Comment (bobmyhill) [inline contrib/python/scripts/pyvista_2d_example.py:99]: Might be good to format the file with black (https://black.readthedocs.io/en/stable/)

**Review by bobmyhill — COMMENTED** [review]
Thanks @naliboff , this looks good. A couple of super minor comments and then this is good to go. Maybe if someone finds a better solution at some point we can revisit.

**Comment by bobmyhill** [comment]
Oh, also, it would be great to give instructions on how you generated this vtk

File: contrib/python/scripts/pyvista_2d_example.py (hunk 2 above)
Comment (naliboff) [reply contrib/python/scripts/pyvista_2d_example.py:99]: Done!

**Comment by naliboff** [comment]
@bobmyhill - I think all the comments have been addressed, thanks!

**Review by bobmyhill — APPROVED** [review]
Great, thanks! All good for me!

**Pull request merged**